modules/rpc/yarpc_test.go

```diff
@@ -36,30 +35,36 @@ func TestRegisterDispatcher_OK(t *testing.T) {
 	RegisterDispatcher(defaultYARPCDispatcher)
 }
 
+func TestRegisterStarter_OK(t *testing.T) {
+	t.Parallel()
+	RegisterStarter(defaultYARPCStarter)
```

Comment:
That's not really my point.

My point is that if there is another parallel test with a custom starter, you have just created a potential for a flappy test, depending on which one starts first, right?